src/js/components/visualization/ChartComponent.jsx

```diff
@@ -83,15 +85,13 @@ export class ChartComponent extends Component {
         data={chartData}
         drilldown={drillDownFieldName != null}
         options={options}
-        ref='chart'
+        ref={(ref) => (this.chart = ref)}
```

Comment:
I am fine with doing `ref={(ref) => (this._chart = ref)}`. so callbacks and underscores.